src/components/Filters/Filters.tsx

```diff
@@ -204,7 +257,7 @@ function Filters({ partners, setFilteredPartners }: Props) {
           <Button onClick={handleClearFilters} basic secondary>
             <FormattedMessage id="clear" />
           </Button>
-          <Button onClick={handleApplyFilters} basic primary>
+          <Button onClick={() => handleApplyFilters()} basic primary>
```

Comment:
`{handleApplyFilters}` should work here

Reply:
No because `handleApplyFilters` receives an optional argument :(

<img width="1045" alt="image" src="https://user-images.githubusercontent.com/45410089/196488851-aed73561-8a58-4be0-a4a7-b5e557c54113.png">